trace-server/org.eclipse.tracecompass.incubator.trace.server.jersey.rest.core.tests/src/org/eclipse/tracecompass/incubator/trace/server/jersey/rest/core/tests/services/ConfigurationManagerServiceTest.java

```diff
@@ -388,4 +481,24 @@ private static void validateConfig(ITmfConfiguration config) {
         assertEquals(EXPECTED_CONFIG_DESCRIPTION, config.getDescription());
         assertTrue(config.getParameters().isEmpty());
     }
+
+    @SuppressWarnings("null")
+    private static void validateJsonConfig(ITmfConfiguration config) {
+        assertEquals(EXPECTED_JSON_CONFIG_NAME, config.getName());
+        assertEquals(EXPECTED_JSON_CONFIG_ID, config.getId());
+        assertEquals(CONFIG_WITH_SCHEMA_ANALYSIS_TYPE_ID, config.getSourceTypeId());
+        assertEquals(EXPECTED_JSON_CONFIG_DESCRIPTION, config.getDescription());
+        Map<String, Object> parameters = config.getParameters();
+        assertNotNull(parameters);
+        Map<String, Object> jsonMap = config.getParameters();
+        assertNotNull(jsonMap);
```

Comment:
These two lines just duplicates of the two lines above them?

Reply:
good eye. Done.